### PR TITLE
Add Monetary Committee Charter SNIP

### PR DESCRIPTION
**Pull request by Giri-Aayush**
This SNIP establishes the governance framework for Starknet's Monetary Committee.

Discussion: https://community.starknet.io/t/snip-tbd-starknet-monetary-committee/115064

Changes:
- Add SNIP for Monetary Committee Charter
- Define composition, responsibilities and operational procedures
- Establish parameter management workflows

### CODE REVIEW

File: SNIPS/snip-monetary-committee.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+  - Yearly inflation cap
+  - Withdrawal delay
+2. Sending multi-sig transactions requiring 75% member consensus.
+3. Complying with:
+  - The Code of Conduct of the Starknet governing body.
+  - Any additional internal conflict of interest procedures that the Council may develop from time to time.
+
+#### Availability Commitment
+Members MUST be responsive through the following communication channels with a maximum delay of four (4) days:
+- Phone call - goes through silent mode (installing Opus Genie)
```
Comment (leo-starkware) [inline SNIPS/snip-monetary-committee.md:90]: typo: Opsgenie

File: SNIPS/snip-monetary-committee.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
+---
+snip: TBD
```
Comment (leo-starkware) [inline SNIPS/snip-monetary-committee.md:2]: Can you change the snip number to 27?

File: assets/.DS_Store
Comment (leo-starkware) [inline assets/.DS_Store:1]: Please delete this file

Comment (Giri-Aayush) [reply assets/.DS_Store:1]: Done!

File: SNIPS/snip-monetary-committee.md (hunk 2 above)
Comment (Giri-Aayush) [reply SNIPS/snip-monetary-committee.md]: Done!

File: SNIPS/snip-monetary-committee.md (hunk 1 above)
Comment (Giri-Aayush) [reply SNIPS/snip-monetary-committee.md]: Done!

**Review by leo-starkware — APPROVED** [review]

**Pull request merged**